src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -768,7 +768,22 @@ double dimensionful_spin_magnitude(
   const auto potentials =
       get_normalized_spin_potentials(smallest_eigenvectors, ylm, area_element);
 
-  return get_spin_magnitude(potentials, spin_function, area_element, ylm);
+  *result = get_spin_magnitude(potentials, spin_function, area_element, ylm);
```

Comment:
is there a not::null version of get_spin_magnitude, so you can just do `get_spin_magnitude(result, potentials, spin_function, area_element, ylm);`

Reply:
There isn't a not::null version of `get_spin_magnitude` , do I have to make one? @geoffrey4444